great_expectations/datasource/data_connector/util.py

```diff
@@ -9,21 +9,29 @@
 from pathlib import Path
 from typing import Any, Dict, List, Optional, Tuple, Union
 
-from azure.core.paging import ItemPaged
-from azure.storage.blob import (
-    BlobPrefix,
-    BlobProperties,
-    BlobServiceClient,
-    ContainerClient,
-)
-
 from great_expectations.core.batch import BatchDefinition, BatchRequestBase
 from great_expectations.core.id_dict import IDDict
 from great_expectations.data_context.util import instantiate_class_from_config
 from great_expectations.datasource.data_connector.sorter import Sorter
 
 logger = logging.getLogger(__name__)
 
+try:
+    from azure.storage.blob import (
+        BlobPrefix,
+        BlobServiceClient,
+        ContainerClient,
+    )
+except ImportError:
+    ItemPaged = None
+    BlobPrefix = None
+    BlobProperties = None
+    BlobServiceClient = None
+    ContainerClient = None
+    logger.debug(
```

Comment:
I think the items removed from the `try` clause can also be removed here!

Reply:
But in the interest of getting this out ASAP, maybe we can change this after.

Reply:
fixed, once and for all (tm)